packages/playground/src/weblets/profile_manager.vue

```diff
@@ -722,7 +728,12 @@ async function activateAccount() {
   activatingAccount.value = true;
   activating.value = true;
   try {
-    await activateAccountAndCreateTwin(mnemonic.value);
+    const mnemonicValue = validateMnemonic(mnemonic.value)
+      ? mnemonic.value
+      : mnemonic.value.length === 66
+      ? mnemonic.value
+      : `0x${mnemonic.value}`;
```

Comment:
should we use if/else if ? i think it will be more readable

Reply:
I think it's good, less code, and the results depend on the previous one. it's fine.
@AhmedHanafy725 what do you think ?

Reply:
it's fine but what confuses me is the variable name, it can be mnemonic or hex seed